src/api/routes/conferenceSupervisor.ts

```diff
@@ -10,7 +10,70 @@ import Elysia, { t } from 'elysia';
 
 export const conferenceSupervisor = new Elysia()
 	.use(permissionsPlugin)
-	.use(CRUDMaker.getAll('conferenceSupervisor'))
+	.get(
+		'/conferenceSupervisor',
+		async ({ permissions, query }) => {
+			const user = await permissions.mustBeLoggedIn();
+
+			return await db.conferenceSupervisor.findMany({
+				where: {
+					OR: [
+						{
+							delegations: {
+								some: {
+									id: query.delegationId
+								}
+							}
+						},
+						{
+							conference: {
+								id: query.conferenceId
+							}
+						}
+					]
+				},
+				include: {
+					user: {
+						select: {
+							id: true,
+							given_name: true,
+							family_name: true
+						}
+					},
+					_count: {
+						select: {
+							delegations: true
+						}
+					}
+				}
+			});
+		},
+		{
+			query: t.Optional(
```

Comment:
When all fields on an object can be optional you can use the t.Partial() helper to achieve that!